### DISCUSSION TITLE
Self equip

### DISCUSSION THREAD

**Discussion by vo1ced** [isid0re/SoloLeveling#344] [Q&A]
Hi! Coming back from the d2jsp times (the bot times, not what it seems to have evolved into since I last logged in 15 years ago or so.) So this is awesome, love the script, I can really appreciate how much work has been put into it. Was really cool to sit back until lvl 25 with Hammerdin.

Now I started a Javazon and thought I'd help the bot by pre-equipping it some low level set items etc. I see it sold them instantly and equipped something much worse.

My first question is - once I reach the right level, can I safely equip it with Titans or may it sell those too? Second question is, is there somewhere listed in the configs what items it is striving for, and what I can equip that will be superior and kept? 

Last but not least - much appreciated that there is a Hardcore (ladder) profile <3 Keep up the good work.

**Comment by isid0re** [comment]
I wouldnt equip interim items. Those will be lost in the course of events. You can look at the `/libs/sololeveling/buildfiles` folder and find the respective finalbuild. Within said file, will list the finalgear autoequip items. 

If you try and gear the bot up before it's ready you WILL trigger a respec on the bot and it will be unleveled and broken. As people have tried that before with their bots.

I would advise you to just let the bot level up till it's at a level u are happy with. Then take it off sololeveling, fill in the gear gaps yourself ,and then use stock kolbot.

**Reply by vo1ced** [reply]
That's all right, I think it will be even more interesting this way. Also, it did Stealth in no time! Like I said, impressive and good work. Thank you.
